code/src/main/java/com/google/sps/data/BusinessNumber.java

```diff
@@ -0,0 +1,36 @@
+package com.google.sps.data;
```

Comment:
Let's name this package "storage", rather than "data".

Reply:
I changed everything from package com.google.sps.data to  package com.google.sps.storage. I also changed the file name to reflect this